### PR TITLE
API Integration for challenges

### PR DESCRIPTION
**Pull request by Ankita2002-Fr**
@whyDontI  @sumitd94 

Fetched data from challenges API. Please review the PR.

https://github.com/Real-Dev-Squad/website-status/commit/9c9533cf029d189627d69188c781b18157eab148

### CODE REVIEW

File: src/pages/challenges.tsx [hunk 1]
```diff
@@ -1,17 +1,52 @@
...
-    <Head title="Challenges" />
-    <div>
-      <Section content={ChallengesJson} screen={CHALLENGES_SCREEN} />
-    </div>
-  </Layout>
-);
+const CHALLENGES_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/challenges`;
+
+const Challenges: FC = () => {
+  let challenges:any[] = [];
```
Comment (sumitd94) [inline src/pages/challenges.tsx:12]: why are we giving this `any`? don't we know yet what type of data will be stored in the array?

File: src/pages/challenges.tsx [hunk 2]
```diff
@@ -1,17 +1,52 @@
...
+            ) : (
+              <>
+                {
+                 filterChallenge.length > 0
+                   ? (
+                     <div>
+                       <Section content={filterChallenge} screen={CHALLENGES_SCREEN} />
+                     </div>
+                   )
+                   : (!error && 'No Tasks Found')
```
Comment (sumitd94) [inline src/pages/challenges.tsx:42]: NIT: shouldn't this be `No Challenges found`?

**Review by sumitd94 — APPROVED** [review]
Few questions, rest all looks good here.

**Comment by whyDontI** [comment]
Duplicate of - https://github.com/Real-Dev-Squad/website-status/pull/123

**Pull request closed without merging**